quickwit/quickwit-control-plane/src/lib.rs

```diff
@@ -44,113 +34,5 @@ pub struct IndexerNodeInfo {
 
 pub type IndexerPool = Pool<String, IndexerNodeInfo>;
 
-/// Subscribes to various metastore events and forwards them to the control plane using the inner
-/// client. The actual subscriptions are set up in `quickwit-serve`.
-#[derive(Debug, Clone)]
-pub struct ControlPlaneEventSubscriber(ControlPlaneServiceClient);
```

Comment:
Arg. I'm adding two callbacks in my PR :D

Reply:
We can readd whatever is needed.